workloads/kube-burner/common.sh

```diff
@@ -70,13 +70,20 @@ run_workload() {
   CMD="timeout ${JOB_TIMEOUT} ${KUBE_DIR}/kube-burner init --uuid=${UUID} -c $(basename ${WORKLOAD_TEMPLATE}) --log-level=${LOG_LEVEL}"
   # When metrics or alerting are enabled we have to pass the prometheus URL to the cmd
   if [[ ${INDEXING} == "true" ]] || [[ ${PLATFORM_ALERTS} == "true" ]] ; then
-    CMD+=" -u=${PROM_URL} -t ${PROM_TOKEN}"
+    if [[ -n ${PROM_URL} ]] && [[ ${PROM_URL} != "None" ]]; then
```

Comment:
If we are already in the indexing if we need either A) a prom url or B) the metrics endpoint. these two new ifs should be condensed down to 1 if/else IMHO and then document the precedence.

Reply:
Updated as suggested. Metrics endpoints option has precedence over prom URL. Thanks.